actionpack/lib/action_controller/metal/http_authentication.rb

```diff
@@ -74,6 +74,8 @@ module ClassMethods
           #
           # See ActionController::HttpAuthentication::Basic for example usage.
           def http_basic_authenticate_with(name:, password:, realm: nil, **options)
+            raise ArgumentError, "Expected name: to be a String, got #{name.class}" unless name.is_a?(String)
+            raise ArgumentError, "Expected password: to be a String, got #{password.class}" unless name.is_a?(String)
```

Comment:
I assume this is supposed to be `unless password.is_a?(String)`?

Reply:
🤦 yes you are right. I'll fix directly in `main`. Thanks!